### PR TITLE
feat(HACBS-2127): add automated to release status

### PR DESCRIPTION
**Pull request by johnbieren**
@davidmogar not sure if you want me to create a `r.markAutomated` function that sets the status or just set the status directly in integration-service. Figured you'd have an opinion on it so only added the field until I hear it

### CODE REVIEW

File: api/v1alpha1/release_types.go [hunk 1]
```diff
@@ -130,6 +130,9 @@ type ProcessingInfo struct {
 
 // ValidationInfo defines the observed state of the release validation.
 type ValidationInfo struct {
+	// Automated indicates whether the Release was created by the integration service
+	Automated bool `json:"automated,omitempty"`
```
Comment (davidmogar) [inline api/v1alpha1/release_types.go:134]: How is this related to validation info? It shouldn't be here. Instead, this should probably be a top level field.

File: api/v1alpha1/release_types.go [hunk 2]
```diff
@@ -130,6 +130,9 @@ type ProcessingInfo struct {
 
 // ValidationInfo defines the observed state of the release validation.
 type ValidationInfo struct {
+	// Automated indicates whether the Release was created by the integration service
```
Comment (davidmogar) [inline api/v1alpha1/release_types.go:133]: I don't really like to mention services in the comments in case the name changes or those services get split (e.g. has and build services). What about:
```suggestion
	// Automated indicates whether the Release was created as part of an automated process or manually by an end-user
```

File: api/v1alpha1/release_types.go (hunk 1 above)
Comment (johnbieren) [reply api/v1alpha1/release_types.go:134]: Ack

**Review by davidmogar — APPROVED** [review]
Answering to your question, the `Automated`  field can be modified directly from the operation that will process the integration label.

**Review by davidmogar — APPROVED** [review]
